src/tess_asteroids/movingtpf.py

```diff
@@ -1238,25 +1486,8 @@ def _save_tpf(self, file_name: Optional[str] = None, save_loc: str = ""):
                 "Must run `get_data()`, `reshape_data()`, `create_pixel_quality()`, `background_correction()` and `create_aperture()` before saving TPF."
             )
 
-        # Create default file name
-        if file_name is None:
-            file_name = "tess-{0}-s{1:04}-{2}-{3}-shape{4}x{5}-moving_tp.fits".format(
-                str(self.target).replace(" ", ""),
-                self.sector,
-                self.camera,
-                self.ccd,
-                *self.shape,
-            )
-
-        if not file_name.endswith(".fits"):
-            raise ValueError(
-                "`file_name` must be a .fits file. Not `{0}`".format(file_name)
-            )
-        if len(save_loc) > 0 and not save_loc.endswith("/"):
-            save_loc += "/"
-
         # Compute WCS header
-        wcs_header = self._make_wcs_header()
+        wcs_header = make_wcs_header(self.shape)
```

Comment:
The problem is that the average WCS uses reference pixels outside the bounds of the TPF. This will mean the WCS cannot be applied to the TPF pixel data in a straightforward way. I think it could be more confusing to use the average WCS than a dummy one. It's something we can look at improving in the future, but for this PR I think we should keep the dummy WCS (especially seeing as the file contains RA,Dec of the target).